automation/taskcluster/release/fetch-bintray-api-key.py

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Do we need this file?

Reply:
No, so I won't land it -- but it's what we'd want to publish Android libraries in release tasks.  So we'll use it to get the Logins API publishing flow set up.  But it can wait until then.